tests/src/Kernel/MultilingualBlockTest.php

```diff
@@ -87,6 +87,7 @@ public function testBlockRendering(): void {
     $link = $crawler->filter('div.language-switcher > a');
     $this->assertSame('English', trim($link->text()));
     $this->assertSame('#', $link->attr('href'));
+    $this->assertSame('Change language. Current language is English.', $link->attr('aria-label'));
```

Comment:
set another language and assert the language changes

Reply:
added